djangocms_page_meta/utils.py

```diff
@@ -103,8 +97,13 @@ def get_page_meta(page, language):
             meta.twitter_type = pagemeta.twitter_type
             meta.twitter_site = pagemeta.twitter_site
             meta.twitter_author = pagemeta.twitter_author
-            meta.gplus_type = pagemeta.gplus_type
-            meta.gplus_author = pagemeta.gplus_author
+            meta.schemaorg_type = pagemeta.schemaorg_type
+            if page.publication_date:
```

Comment:
we don't need this to be moved inside the `try / except` as they are values taken from the current page, which are always available. By setting them outside the `try / except` we can always add this information even if a page meta is not created